<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-internet">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Is leap</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Next day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Previous day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Week number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Till your birthday</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="closefile_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close file</string>
    </property>
   </widget>
   <widget class="QLabel" name="coorectday_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>255</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Correct date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coorectdaydate_line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>335</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="coorectdaynum_line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>295</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="openfile_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>430</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createfile_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>465</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Create file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="adddate_line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="adddate_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>85</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dateofbirthday_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Change birhday</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateofbirthday_line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>165</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="adddate_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add date</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateofbirthday_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>125</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Date of birthday</string>
    </property>
   </widget>
   <widget class="QPushButton" name="coorectday_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Date change</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
